### DISCUSSION TITLE
Difficulty of reading mixed languages

### DISCUSSION THREAD

**Discussion by rzasharifi** [spacecowboy/Feeder#309] [General]
As you can see, there is a disarrangment in text which in it has both Persian(RTL) and English(Ltr) languages therefore it's not easy to read it! However in the site you can read it without any distractions. 

The source [news](https://dgto.ir/3gvp)

![Screenshot_2024-06-15-13-56-32-150_com nononsenseapps feeder-edit](https://github.com/spacecowboy/Feeder/assets/64254187/4ba0cca2-7494-4c3c-adc6-bf0f0ea55262)
![IMG_20240615_141351](https://github.com/spacecowboy/Feeder/assets/64254187/c7419328-1e0b-4993-89b0-579c70e7bece)

**Comment by spacecowboy** [comment]
The problem is probably that it starts with `OpenAI`. I think that makes the algorithm think it's LTR text.

**Reply by rzasharifi** [reply]
![IMG_20240627_113406](https://github.com/spacecowboy/Feeder/assets/64254187/684340e1-537a-4dba-8311-e287b76f8724)
V. 2.6.28 the problem still exists!

**Reply by rzasharifi** [reply]
Hi there. I hope you are doing great. In v. 2.6.29 the problem doesn't solved. Could you please put this in the high priority?

**Reply by spacecowboy** [reply]
It will be mentioned in the changelog when/if resolved.

**Comment by AI-IMED** [comment]
![٢٠٢٤٠٦١٧-٢٠٥٨٣٠](https://github.com/spacecowboy/Feeder/assets/38974362/a7ac5e60-f7a2-41e1-905e-6ce7404c189f)

The solution is to adopt the **anyRtl** algorithm.

**Reply by AI-IMED** [reply]
The strange thing is that I found the old versions of the application, especially versions from 1.8.7 to 1.13.5, it supported all RTL & LTR scenarios in a very wonderful way !!!
